src/components/MobileFooter/MobileFooter.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import FacebookSVG from '../../assets/svgs/facebook-logo-dark-theme.svg';
+import GithubSVG from '../../assets/svgs/github-logo-dark-theme.svg';
+import { FACEBOOK_LINK, GITHUB_LINK } from '../../utils/constants/links';
+import {
+  containerGridDark,
+  containerLinks,
+  mainContainerDark,
+} from './MobileFooter.module.scss';
+export const MobileFooter = () => (
+  <div className={mainContainerDark}>
+    <div className={containerGridDark}>
+      <div>
+        <h2>
+          Koło naukowe i::team <br />
+          Uniwersytetu Ekonomicznego <br />w Krakowie
+        </h2>
+      </div>
+      <div>
+        <h3>Zarząd:</h3>
+        <p>zarzad@example.com</p>
+      </div>
+      <div>
+        <p>
+          Rakowicka 27, 31-510 <br />
+          Kraków <br />
+          Biblioteka Główna p.432 <br />
+          kniuex@example.com
+        </p>
+      </div>
+      <div>
+        <h3>Opiekun Koła:</h3>
+        <p>
+          mgr Katarzyna Wójcik <br />
+          katwoj@fakemail.com
+        </p>
+      </div>
+      <div></div>
+      <div>
+        <h3>Znajdziesz nas na:</h3>
+        <div className={containerLinks}>
+          <a href={GITHUB_LINK}>
+            <GithubSVG />
+          </a>
+          <a href={FACEBOOK_LINK}>
+            <FacebookSVG />
```

Comment:
tego chyba brakuje

Reply:
albo jakis bug jest bo się nie wyświetla (są dwa pliki i dwa zaciaga zamiast 1)
![image](https://github.com/UEK-i-team/team-homepage/assets/49104061/d3da9894-0115-4030-ad0a-30e1c0b98c71)

Reply:
To byl problem z id, które figma przypisuje przy eksportowaniu svg. Naprawione